### PR TITLE
Add spider for bmw_es

### PR DESCRIPTION
**Pull request by abbensid2**

### CODE REVIEW

File: locations/spiders/bmw_es.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        "brand": "bmw",
+        "brand_wikidata": "Q26678",
+    }
+    allowed_domains = ["bmw.es"]
+    start_urls = [
+        "https://www.bmw.es/c2b-localsearch/services/api/v4/clients/BMWSTAGE2_DLO/ES/pois?brand=BMW_BMWI_BMWM&cached=off&category=BM&country=ES&language=es&lat=0&lng=0&maxResults=700&showAll=true&unit=km"
+    ]
+
+    def parse(self, response):
+        results = response.json()["data"]["pois"]
```
Comment (iandees) [inline locations/spiders/bmw_es.py:18]: It looks like this comes back as XML, not JSON.

Comment (iandees) [reply locations/spiders/bmw_es.py:18]: Maybe try sending an `accept: application/json` header in the initial request?

Comment (Romeo-PHILLIPS) [reply locations/spiders/bmw_es.py:18]: Yea same result

File: locations/spiders/bmw_es.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import scrapy
+import xmltodict
+
+from locations.dict_parser import DictParser
+
+
+class BeerBmwEsSpider(scrapy.Spider):
+    name = "bmw_es"
+    item_attributes = {
+        "brand": "bmw",
```
Comment (iandees) [inline locations/spiders/bmw_es.py:12]:
```suggestion
        "brand": "BMW",
```

File: locations/spiders/bmw_es.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        "brand_wikidata": "Q26678",
+    }
+    allowed_domains = ["bmw.es"]
+    start_urls = [
+        "https://www.bmw.es/c2b-localsearch/services/api/v4/clients/BMWSTAGE2_DLO/ES/pois?brand=BMW_BMWI_BMWM&cached=off&category=BM&country=ES&language=es&lat=0&lng=0&maxResults=700&showAll=true&unit=km"
+    ]
+
+    def parse(self, response):
+        data_dict = xmltodict.parse(response.text)
+        json_data = json.loads(json.dumps(data_dict))
```
Comment (iandees) [inline locations/spiders/bmw_es.py:22]: There's no need to pass the data through json.loads/json.dumps.

```suggestion
        json_data = xmltodict.parse(response.text)
```

**Pull request merged**